libraries/AP_Mount/AP_Mount_STorM32_native.cpp

```diff
@@ -1,26 +1,20 @@
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Common/AP_Common.h>
+#include <AP_AHRS/AP_AHRS.h>
+#include <AP_SerialManager/AP_SerialManager.h>
+#include <AP_GPS/AP_GPS.h>
+#include <GCS_MAVLink/GCS.h>
+#include <RC_Channel/RC_Channel.h>
 #include "AP_Mount_STorM32_native.h"
 
 extern const AP_HAL::HAL& hal;
 
 AP_Mount_STorM32_native::AP_Mount_STorM32_native(AP_Mount &frontend, AP_Mount::mount_state &state, uint8_t instance) :
-    AP_Mount_Backend(frontend, state, instance),
-    _initialised(false),
-    _armed(false),
-    _send_armeddisarmed(false)
+    AP_Mount_Backend(frontend, state, instance)
 {
-    _uart = nullptr;
     _mount_type = AP_Mount::Mount_Type_None; //the mount type will be determined in init()
 
-    _task_time_last = 0;
-    _task_counter = TASK_SLOT0;
-
     _bitmask = SEND_STORM32LINK_V2 | SEND_CMD_SETINPUTS | SEND_CMD_DOCAMERA;
-
-    _status.pitch_deg = _status.roll_deg = _status.yaw_deg = 0.0f;
-    _status_updated = false;
-
-    _target_to_send = false;
-    _target_mode_last = MAV_MOUNT_MODE_RETRACT;
```

Comment:
Removing this last one seems wrong as I don't see it added to class declaration.